density/templates/predict.html

```diff
@@ -9,20 +9,98 @@
 
 <div class="container">
   <div class="col-12"  style="text-align:center;">
-    <h4 style="margin-top: 30px;">Below is the predicted capacity of each building for today: </h4>
+    <h4 style="margin-top: 30px;">Below is the predicted capacity of each building for today.<br><br></h4>
   </div>
 </div>
 
+<div class="container">
+        <div class="searchContainer">
+            <input type="text" class="searchTerm" id="search" onkeyup="searchLocations()" autocomplete= "off">
+            <button type="submit" class="searchButton">
+                <i class="fa fa-search"></i>
+            </button>
+        </div>
+    </div>
+
+    <div class="container">
+        <div class = "btnContainer">
+          <div id="locationBtnContainer" align="left">
+              <input type="radio"  name="location-filter" id="library" value="library" onclick = "searchLocations()"/> <label for="library"> Library </label>
+              <input type="radio"  name="location-filter" id="center" value="center" onclick = "searchLocations()"/> <label for="center"> Student Center </label>
+              <input type="radio"  name="location-filter" id="dining" value="dining" onclick = "searchLocations()"/> <label for="dining"> Dining Hall </label>
+              <button class="clear" onclick="clearFilter('location-filter')">(x)</button>
+          </div>
+
+          <div id="openBtnContainer" align="right">
+              <input type="radio" class="filterTab" name="open-filter" id="open" value="True" onclick = "searchLocations()"/> <label for="open"> Open Now </label>
+              <input type="radio" class="filterTab" name="open-filter" id="closed" value="False" onclick = "searchLocations()"/> <label for="closed">Closed </label>
+              <button class="clear" onclick="clearFilter('open-filter')">(x)</button>
+          </div>
+        </div>
+    </div>
```

Comment:
If this is just a repeat of above, then we should refactor this into a Jinja2 macro.